modules/apps/remote-app/remote-app-service/service.xml

```diff
@@ -19,8 +19,12 @@
 
 		<!-- Other fields -->
 
+		<column name="customElementCSSURLs" type="String" uad-nonanonymizable="true" />
+		<column name="customElementHTMLElementName" type="String" uad-nonanonymizable="true" />
+		<column name="customElementURLs" type="String" uad-nonanonymizable="true" />
```

Comment:
JSURLs?

Reply:
:shrug: If it is `iframeURL`, not `iframeHTMLURL`, then it shouldn't be `customElementJSURLs`, I guess

Reply:
Yep... my worries are because you have `customElementCSSURLs` and `customElementURLs`